vividus-to-zephyr-for-jira-exporter/src/main/resources/org/vividus/jira/spring.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans
+        xmlns="http://www.springframework.org/schema/beans"
+        xmlns:util="http://www.springframework.org/schema/util"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:context="http://www.springframework.org/schema/context"
+        xsi:schemaLocation="
+        http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+        default-lazy-init="true">
+
+    <bean id="jiraFacade" class="org.vividus.facade.jira.JiraFacade">
+        <property name="client" ref="jiraClient" />
+        <property name="jiraConfiguration">
+            <bean class="org.vividus.facade.jira.JiraConfiguration">
+                <property name="username" value="${jira.username}" />
+                <property name="password" value="${jira.password}" />
+                <property name="endpoint" value="${jira.endpoint}" />
+            </bean>
+        </property>
+    </bean>
+
+    <bean id="zephyrFacade" class="org.vividus.facade.zephyr.ZephyrFacade">
+        <property name="client" ref="jiraClient" />
+        <property name="jiraConfiguration">
+            <bean class="org.vividus.facade.jira.JiraConfiguration">
+                <property name="username" value="${jira.username}" />
+                <property name="password" value="${jira.password}" />
+                <property name="endpoint" value="${jira.endpoint}" />
+            </bean>
```

Comment:
move `jiraConfiguration` to a separate bean and reuse it in `jiraFacade` and `zephyrFacade`